source/renderer/app/actions/assets-actions.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import Action from './lib/Action';
+import type { WalletSummaryAsset } from '../api/assets/types';
+
+// ======= ADDRESSES ACTIONS =======
+
+export default class AddressesActions {
```

Comment:
Can you rename this to the `AssetsActions`

Reply:
Done ✓